lib/reactotron-mst/src/reactotron-mst.ts

```diff
@@ -263,10 +284,15 @@ export function mst(opts: MstPluginOptions = {}) {
      *
      * @param command A reactotron command.
      */
-    function backup(command: any) {
-      const trackedNode = trackedNodes[command.mstNodeName || "default"]
+    function backup(command: Command<"state.backup.request">) {
+      const trackedNode =
+        trackedNodes[
+          "mstNodeName" in command && typeof command.mstNodeName === "string"
```

Comment:
No client sends a command with this key. So this boolean always evaluates to 'default'. Future TODO would be to figure out what the actually desired behavior is here